daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -161,7 +161,7 @@ class Engine(val config: EngineConfig = EngineConfig.Stable) {
     } yield (tx, meta)
 
   def replay(
-      submitter: Party,
+      submitters: Set[Party],
```

Comment:
@rautenrieth-da You've mentioned that we should use here `List[Party]` but `Set[Party]` seems to be used across the `Engine`, so I used the latter. Is that ok for you?

Reply:
Yes, IMO the interface should be consistent with the rest of `Engine`.